app/src/app/core/components/top-menu/top-menu.component.ts

```diff
@@ -1,11 +1,25 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
+import { AuthSelectors } from '@data/auth';
+import { Store } from '@ngrx/store';
+import { Observable, Subscription } from 'rxjs';
 
 @Component({
   selector: 'core-top-menu-component',
   templateUrl: './top-menu.component.html',
   styleUrls: ['./top-menu.component.scss']
 })
 
-export class TopMenuComponent {
+export class TopMenuComponent implements OnInit{
+  public isAdmin;
+  public baseUrl = window.location.origin;
+  constructor(
+    private store: Store<any>
```

Comment:
Detta är en component och inte en container (page eller i detta fallet layout), den bör inte ha access till store utan bör enbart jobba med input.

Reply:
Done. Moved Store to default.component and full.component